Sources/NIO/System.swift

```diff
@@ -377,6 +377,8 @@ internal enum Posix {
                         written = 0
                     }
                     return result
+                #else
+                    let badOS = { fatalError("unsupported OS") }()
```

Comment:
Do we need the self-invoking function? I think just the `fatalError` itself would be fine.

Reply:
Yeah, right, just copied what the top-level was doing.